corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -58,6 +59,12 @@ def add_referenced_instances(self, entry, details_by_id, relevance_by_menu, menu
         xpaths.discard(None)
         instances, unknown_instance_ids = EntryInstances.get_required_instances(xpaths)
 
+        if toggles.CUSTOM_CALENDAR_FIXTURE.enabled(self.app.domain):
```

Comment:
Initially wanted to add something like https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml/post_process/instances.py#L121 but didn't to pollute those classes/functions with knowledge about the app or domain